Gemfile

```diff
@@ -129,9 +129,9 @@ group :test, :development do
   gem 'atomic'
   gem 'awesome_print'
   gem 'capybara', '>= 2.18.0' # 3.0 requires nokogiri 1.8
-  gem 'database_cleaner', '0.7.1', require: false
-  gem "factory_bot_rails", require: false
-  gem 'fuubar', '~> 2.5.0'
+  gem 'database_cleaner', require: false
+  gem "factory_bot_rails", '4.8.2', require: false
```

Comment:
Why this particular version? https://rubygems.org/gems/factory_bot_rails/versions/4.10.0 seems to be the latest version supporting Rails 4. The next one requires `railties >= 4.2.0`.

Reply:
4.10.0 works fine :+1: